webpack.config.js

```diff
@@ -0,0 +1,64 @@
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const path = require('path');
+const pkg = require('./package.json');
+const webpack = require('webpack');
+
+module.exports = {
+	devtool: 'source-map',
+	entry: './src/index.js',
+	mode: 'production',
+	optimization: {
+		minimize: false
```

Comment:
If `grunt`'s no longer uglifying, then what is for `.min`? Or are you just not generating one?

Reply:
I'll eventually let `minimize` be `true`.  I left it as false for now because I wanted to see the output :-).